code/core/src/handlers/AssetHandler.java

```diff
@@ -0,0 +1,54 @@
+package handlers;
+
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.assets.loaders.FileHandleResolver;
+import com.badlogic.gdx.assets.loaders.resolvers.InternalFileHandleResolver;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeType;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGeneratorLoader;
+import com.badlogic.gdx.graphics.g2d.freetype.FreetypeFontLoader;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+/**
+ * Class that handles the loading of all assets
+ */
+public class AssetHandler {
+
+    public AssetManager manager = new AssetManager();
```

Comment:
Can be static

Reply:
The documentation says it shouldn't be static as it might cause issues, so I leave it as is. https://github.com/libgdx/libgdx/wiki/Managing-your-assets